### PR TITLE
feat(mobile): new user mobile flow 

### PR DESCRIPTION
**Pull request by baymax-codecare**
**What this PR does** 📖
 new user Userflow for mobile
**Which issue(s) this PR fixes** 🔨
AP-975

**Special notes for reviewers** 🗒️
https://www.figma.com/file/RGLT0NaG7Pin0XTxUaE0JV/New-User-Userflow?node-id=0%3A1
**Additional comments** 🎤

### CODE REVIEW

File: pages/message/index.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    searchResult() {
+      return this.friends.all.filter((user: User) =>
+        user.name.toLowerCase().includes(this.search.toLowerCase()),
+      )
+    },
+  },
+  watch: {
+    search(newValue, oldValue) {
+      if (newValue !== oldValue) {
+        this.selectedAddress = undefined
```
Comment (molimauro) [inline pages/message/index.vue:42]: set empty string

**Comment by phillsatellite** [comment]
Tested: Works a lot better! Small little fix the add friends button seems to not work (see screen recording)

https://user-images.githubusercontent.com/93608357/158426445-6396d587-fb4b-45ae-b3a6-96e4341cd220.mov

File: pages/message/index.vue (hunk 1 above)
Comment (baymax-codecare) [reply pages/message/index.vue]: fixed @molimauro , now it's using ''

**Comment by baymax-codecare** [comment]
> Tested: Works a lot better! Small little fix the add friends button seems to not work (see screen recording)
> 
>  Screen.Recording.2022-03-15.at.12.31.56.PM.mov



> Tested: Works a lot better! Small little fix the add friends button seems to not work (see screen recording)
> 
>  Screen.Recording.2022-03-15.at.12.31.56.PM.mov

fixed not navigating issue. 👍

**Comment by phillsatellite** [comment]
Tested: Dm'd David L but wanted to comment on here as well was going over this with liz when she had mentioned that user shouldn't be able to swipe like that swiping is for Forward/backward only, to navigate to a page user should have to use the buttons/icons

https://user-images.githubusercontent.com/93608357/158453577-d9239d2e-95c0-4cc5-ab12-b6e9ac0d4a37.mov

**Comment by WanderingHogan** [comment]
Since the designs/plans for this changed, please re-open this PR when you are ready for us to review it again :)

**Pull request closed without merging**